template/bacardi_cpp.njk

```diff
@@ -23,10 +23,11 @@
 #include "{{interface.idl_dir_name}}/{{interface.name | snakecase}}_bridge.h"
 {% endfor %}
 
-void Init(Napi::Env env, Napi::Object exports, Napi::Object module) {
+Napi::Object Init(Napi::Env env, Napi::Object exports) {
```

Comment:
Is it one of changes to Node.js 8.9.1? :D

Reply:
@nadongguri Yeah, this is a change of wrapper API.